tests/org.jboss.tools.central.ui.bot.test/pom.xml

```diff
@@ -14,13 +14,7 @@
 	<packaging>eclipse-test-plugin</packaging>
 
 	<properties>
-		<jbosstools.test.wildfly.home>${requirementsDirectory}/wildfly-10.0.0.Final</jbosstools.test.wildfly.home>
-		<wildflySystemProperties></wildflySystemProperties>
-		<eapSystemProperties></eapSystemProperties>
-		<jbosstools.test.jboss-eap-6.4.home>${requirementsDirectory}/jboss-eap-6.4</jbosstools.test.jboss-eap-6.4.home>
-		<jbosstools.test.jboss-eap-7.0.home>${requirementsDirectory}/jboss-eap-7.0</jbosstools.test.jboss-eap-7.0.home>
-		<rd.config>${project.build.directory}/classes/server_properties</rd.config>
```

Comment:
so you dont use servers at all ? If yes, than you can also remove https://github.com/jbosstools/jbosstools-integration-tests/tree/master/tests/org.jboss.tools.central.ui.bot.test/resources/server_properties